### ISSUE TITLE
Add a close icon to the room settings page

### ISSUE THREAD

**Issue by SofienM** [openfun/jitsi-magnify#175]
## Feature Request

**Problem**:
Currently, when a user wants to leave the room's settings page, he/she has to click on the browser back button or on the "Salles" link in the breadcrumb. While this works well, adding a more explicit way to get back could improve the UX.

**Solution**
I propose to add a cross btn on the right side of the page. The screenshots were done with a `grommet` icon as it is already installed. 

![Screenshot from 2023-04-03 17-14-27](https://user-images.githubusercontent.com/45047538/229554390-25aca5a5-77e8-4dd2-8fb5-e2b89f6d7cc6.png)
![Screenshot from 2023-04-03 17-15-29](https://user-images.githubusercontent.com/45047538/229554416-b6ab5f7c-57ec-47c5-a08f-c0604886665a.png)

**Do you want to work on it through a Pull Request?**
Sure !

**Comment by SofienM** [comment]
Seeing how the `Back` button is designed for the User settings page, it might be better to uniformize UX all accross the app. 

**Updated solution**:

The updated solution would be to replace the breadcrumb into a simple "Back" link.
WDYT ?


![image](https://user-images.githubusercontent.com/45047538/230644293-eeba8dae-f2a8-4430-84d7-4ae8680c4d38.png)

**Comment by sampaccoud** [comment]
I agree that a "back" link is better than the close button since this is a normal page and not a modal.
